bin/build

```diff
@@ -117,13 +117,21 @@ EOF
 
 apt_env_dir=$apt_layer/env
 mkdir -p $apt_env_dir
-echo -n "$apt_layer/usr/bin:$PATH" > $apt_env_dir/PATH
-echo -n "$apt_layer/usr/lib/x86_64-linux-gnu:$apt_layer/usr/lib/i386-linux-gnu:$apt_layer/usr/lib" > $apt_env_dir/LD_LIBRARY_PATH
-echo -n "$apt_layer/usr/lib/x86_64-linux-gnu:$apt_layer/usr/lib/i386-linux-gnu:$apt_layer/usr/lib" > $apt_env_dir/LIBRARY_PATH
-echo -n "$apt_layer/usr/include:$apt_layer/usr/include/x86_64-linux-gnu" > $apt_env_dir/INCLUDE_PATH
-cp $apt_env_dir/INCLUDE_PATH $apt_env_dir/CPATH
-cp $apt_env_dir/INCLUDE_PATH $apt_env_dir/CPPPATH
-echo -n "$apt_layer/usr/lib/x86_64-linux-gnu/pkgconfig:$apt_layer/usr/lib/i386-linux-gnu/pkgconfig:$apt_layer/usr/lib/pkgconfig" > $apt_env_dir/PKG_CONFIG_PATH
+echo -n "$apt_layer/usr/bin:$PATH" > $apt_env_dir/PATH.prepend
```

Comment:
Do we still need $PATH here?

Reply:
Yeah I'd think we'd want to be able to shell out to the things included by a binary. For instance, a developer might want to use `convert` from the imagemagick package via a shell call.